src/appengine/handlers/cron/corpus_backup.py

```diff
@@ -66,6 +67,24 @@ def _make_corpus_backup_public(target, corpus_fuzzer_name_override,
         'Failed to mark corpus backup %s public.' % corpus_backup_url)
     return
 
+  filename = (
+      corpus_manager.PUBLIC_BACKUP_TIMESTAMP + os.extsep +
+      corpus_manager.BACKUP_ARCHIVE_FORMAT)
+  public_url = os.path.join(os.path.dirname(corpus_backup_url), filename)
+
+  try:
+    result = storage.copy_blob(corpus_backup_url, public_url)
+    if result:
+      result = storage.set_acl(public_url, 'allUsers')
+  except:
+    result = None
```

Comment:
Why not log_error here instead? That way we log any exception stacks and can avoid this extraneous `result = None` setting and `if not result`.

Reply:
You're right, it's pointless. I've just blindly followed the existing pattern. Fixed this in my diff and two other places above. Thank you!

Reply:
Or actually it had a point, wait a second

Reply:
it seems the intention was to log an error in both cases:
- if there was an exception
- if `return == False`

Reply:
Will think more about this tomorrow. Looks like we can get rid of all try-except blocks inside this function and handle an exception outside of it.

Reply:
yeah, I think now it's much better